### PR TITLE
fix(client-wrapper): Refactor ListStoragePoolClaims

### PR DESCRIPTION
**Pull request by kmjayadeep**
Signed-off-by: Jayadeep KM <kmjayadeep@gmail.com>

### CODE REVIEW

File: utils/k8s/openebscrd.go [hunk 1]
```diff
@@ -21,22 +21,18 @@ import (
 // GetStoragePoolClaim returns the StoragePoolClaim object for given spcName.
 // :return: *openebs_v1.StoragePoolClaim: Pointer to StoragePoolClaim objects.
 func (k8s K8S) GetStoragePoolClaim(spcName string) (*openebs_v1.StoragePoolClaim, error) {
-	spcClient := k8s.OpenebsClientSet.Openebs().StoragePoolClaims()
+	spcClient := k8s.OpenebsClientSet.OpenebsV1alpha1().StoragePoolClaims()
 	return spcClient.Get(spcName, meta_v1.GetOptions{})
 }
 
 // ListStoragePoolClaims returns all the StoragePoolClaim objects
```
Comment (princerachit) [inline utils/k8s/openebscrd.go]: Improve this message. How about `returns an object of StoragePoolClaimList`

File: utils/k8s/openebscrd.go [hunk 2]
```diff
@@ -21,22 +21,18 @@ import (
...
-func (k8s K8S) ListStoragePoolClaims() ([]openebs_v1.StoragePoolClaim, error) {
-	spcCient := k8s.OpenebsClientSet.Openebs().StoragePoolClaims()
-	spcs, err := spcCient.List(meta_v1.ListOptions{})
-	if err != nil {
-		return nil, err
-	}
-	return spcs.Items, nil
+// ListStoragePoolClaims returns an object of StoragePoolClaimList
+func (k8s K8S) ListStoragePoolClaims() (*openebs_v1.StoragePoolClaimList, error) {
+	spcCient := k8s.OpenebsClientSet.OpenebsV1alpha1().StoragePoolClaims()
```
Comment (Akash4927) [inline utils/k8s/openebscrd.go]: it should be spcClient.

Comment (kmjayadeep) [reply utils/k8s/openebscrd.go]: which one?

Comment (sonasingh46) [reply utils/k8s/openebscrd.go]: Line number 30 first word : There is typo : `spcCient`
@kmjayadeep

**Pull request merged**